f8a_worker/data_normalizer.py

```diff
@@ -242,20 +242,25 @@ def _split_keywords(keywords, separator=','):
         return keywords
 
     def _handle_python(self, data):
-        "Handle Python package (setup.py) analysis data"
+        """ setup.py """
```

Comment:
Just a little nitpicking - this is not related to this PR that much, so it would be nice to do it as a separate commit.